src/legacy/Routes.tsx

```diff
@@ -649,7 +652,7 @@ class RouteApp extends React.Component<any, any> {
                     this.setRPCConfig(rpcConfig)
 
                     // set pastel DB thread update timer
-                    if (!app?.isPackaged) {
+                    if (process.env.NODE_ENV === 'production') {
```

Comment:
I think this code is not correct, We need to use `app.isPackaged`. See sample here: https://github.com/pastelnetwork/pastel-electron-wallet/blob/master/src/features/loading/LoadingScreen.tsx (Line 49)